scale: TestMillionCephfsFiles.test_scale_million_cephfs_files failed

Logs: https://ocs4-jenkins-csb-ocsqe.apps.ocp4.prod.psi.redhat.com/job/qe-deploy-ocs-cluster-prod/286/testReport/tests.e2e.scale.test_cephfs_many_files/TestMillionCephfsFiles/test_scale_million_cephfs_files_mgr_/

E           Events:
E             Type     Reason                  Age                    From                     Message
E             ----     ------                  ----                   ----                     -------
E             Normal   Scheduled               5m2s                   default-scheduler        Successfully assigned openshift-storage/csicephfs-demo-podff9d7ae9-085a-42e9-9341-10ed1a00bee8 to compute-0
E             Normal   SuccessfulAttachVolume  5m3s                   attachdetach-controller  AttachVolume.Attach succeeded for volume "pvc-e71343e2-320f-4536-ab8c-cdd557a6f64c"
E             Normal   AddedInterface          4m58s                  multus                   Add eth0 [10.129.2.160/23]
E             Warning  Failed                  4m57s                  kubelet                  Failed to pull image "quay.io/ocsci/nginx:latest": rpc error: code = Unknown desc = can't talk to a V1 docker registry
E             Warning  Failed                  4m17s                  kubelet                  Failed to pull image "quay.io/ocsci/nginx:latest": rpc error: code = Unknown desc = Error reading manifest latest in quay.io/ocsci/nginx: received unexpected HTTP status: 502 Bad Gateway
E             Warning  Failed                  3m40s (x6 over 4m57s)  kubelet                  Error: ImagePullBackOff
E             Warning  Failed                  3m29s (x4 over 4m57s)  kubelet                  Error: ErrImagePull
E             Warning  Failed                  3m29s (x2 over 4m44s)  kubelet                  Failed to pull image "quay.io/ocsci/nginx:latest": rpc error: code = Unknown desc = Requesting bear token: invalid status code from registry 502 (Bad Gateway)
E             Normal   Pulling                 3m29s (x4 over 4m58s)  kubelet                  Pulling image "quay.io/ocsci/nginx:latest"
E             Normal   BackOff                 3m17s (x7 over 4m57s)  kubelet                  Back-off pulling image "quay.io/ocsci/nginx:latest"

ocs_ci/helpers/helpers.py:119: ResourceWrongStatusException

--- a/ocs_ci/helpers/pvc_ops.py
+++ b/ocs_ci/helpers/pvc_ops.py
@@ -65,7 +65,7 @@ def test_create_delete_pvcs(multi_pvc_factory, pod_factory, project=None):
         helpers.wait_for_resource_state(
             resource=pvc_obj,
             state=constants.STATUS_BOUND,
-            timeout=1200,  # Timeout given 5 minutes
+            timeout=1200,  # Timeout given 20 minutes
         )
         pvc_info = pvc_obj.get()
         setattr(pvc_obj, "volume_mode", pvc_info["spec"]["volumeMode"])

--- a/tests/e2e/scale/test_cephfs_many_files.py
+++ b/tests/e2e/scale/test_cephfs_many_files.py
@@ -85,7 +85,7 @@ class MillionFilesOnCephfs(object):
             size=SIZE,
         )
         helpers.wait_for_resource_state(
-            self.cephfs_pvc, constants.STATUS_BOUND, timeout=300
+            self.cephfs_pvc, constants.STATUS_BOUND, timeout=1200
         )
         self.cephfs_pod = helpers.create_pod(
             interface_type=constants.CEPHFILESYSTEM,

